habitat-hitl/habitat_hitl/core/remote_client_state.py

```diff
@@ -222,6 +222,14 @@ def _update_input_state(self, client_states: List[ClientState]) -> None:
                             else delta[1]
                         )
 
+                if "mousePositionDelta" in mouse_json:
+                    pos_delta: List[Any] = mouse_json["mousePositionDelta"]
+                    if len(pos_delta) == 2:
```

Comment:
Other option is empty or None? Or is this just a guard against bad data?

Reply:
This is just a guard (data coming over the network).
It also helps understand what this untyped variable is supposed to mean.